.storybook/preview.js

```diff
@@ -1,45 +1,47 @@
+import { withThemes } from 'storybook-addon-themes';
 import React from 'react';
 import { addDecorator } from '@storybook/react';
-import {
-  ThemeProvider as StorybookThemeProvider,
-  ensure as ensureTheme,
-} from '@storybook/theming';
-import { CSSReset, ThemeProvider } from '@chakra-ui/react';
-import { withThemes } from '@react-theming/storybook-addon';
+import { addons } from '@storybook/addons';
+import { ChakraProvider } from '@chakra-ui/react';
 
+import Theme from './theme';
+// eslint-disable-next-line import/no-unresolved
+import { breakpoints } from '../src/themes/shared/breakpoints';
 import {
-  motoScout24Theme,
-  motoScoutChakraTheme,
   autoScout24Theme,
   autoScoutChakraTheme,
+  motoScout24Theme,
+  motoScoutChakraTheme,
+  // eslint-disable-next-line import/no-unresolved
 } from '../src/themes';
-import { breakpoints } from '../src/themes/shared/breakpoints';
+
+// eslint-disable-next-line import/no-unresolved
 import { Fonts } from '../src/styles/Fonts';
 
-const providerFn = ({ theme = autoScout24Theme, children }) => {
+const ThemeDecorator = (args) => {
+  addons.setConfig({
+    theme: Theme,
+  });
+  const { theme = autoScout24Theme, children } = args;
   return (
-    <StorybookThemeProvider theme={ensureTheme()}>
```

Comment:
@lkappeler as far as I know we need the StorybookThemeProvider. Otherwise, we will have some issues in Canvas:

![Screenshot 2022-08-17 at 09 57 18](https://user-images.githubusercontent.com/37380787/185066369-d24c37bc-21ce-44d1-9c82-49e6e698bde5.png)

Would you mind to redo this part, please 🧸 or be sure that the color works on canvas and the docs as well.
Thank you so much 🌸

Reply:
Ou missed that, that a very unintuitive setup of dependencies 😅

Reply:
there you go https://github.com/smg-automotive/components-pkg/pull/200